lib/simplecov.rb

```diff
@@ -353,7 +353,15 @@ def start_coverage_with_criteria
 
       start_arguments[:eval] = true if coverage_for_eval_enabled?
 
-      Coverage.start(start_arguments) unless Coverage.running?
+      Coverage.start(start_arguments) unless coverage_running?
+    end
+
+    def coverage_running?
+      Coverage.running?
+    rescue NoMethodError
```

Comment:
`Coverage.respond_to? :running?` seems nice than `rescue NoMethodError`, notably because it avoids an exception in case it's not available (and doesn't catch other issues potentially).

Reply:
my thought was that the much more common case will have Coverage.running? implemented so checking for that every time would be slower than rescuing in the rarer case. but this is not a place where slowness matters at all, and I do prefer not to use exceptions for control flow, so agree with you and have updated it.